docs/tags.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: sub-navigation
+order: 7
+title: Adding tags
+description: Use tags to categorise posts and make it easier for readers to browse content on your site.
+---
+
+The GOV.UK Eleventy Plugin makes it easy to add tags to your website. Follow these instructions to enable this feature.
+
+## Create a tags page
+
+To include a list of all tags used on your website, create a page that uses the `tags` layout:
+
+```yaml
+---
+eleventyExcludeFromCollections: true
+layout: tags
+---
+```
+
+You can see an [example of a tag index](../layouts/tags) on this website.
+
+## Create a tag listing page
+
+To generate a page for each tag used on your website, create a page that uses the `tag` layout and paginates `collections.tags` data:
```

Comment:
Might be worth giving some guidance on what to call this file? I don't think the filename actually matters that much (as permalink is overridden) but it can be confusing to know where to put these files which generate multiple files based on a collection...

```suggestion
To generate a page for each tag used on your website, create a page that uses the `tag` layout and paginates `collections.tags` data. Add this content to a `tag.md` file:
```